core/templates/dev/head/editor/ExplorationEditor.js

```diff
@@ -154,7 +154,7 @@ oppia.controller('ExplorationEditor', [
           successCallback();
         }
 
-        stateEditorTutorialFirstTimeService.init(
+        $scope.firstTime = stateEditorTutorialFirstTimeService.init(
```

Comment:
Done. This was already there but I've removed it and there seemed to be no ill effects.